apps/system/js/media_playback.js

```diff
@@ -155,19 +155,30 @@ MediaPlaybackWidget.prototype = {
     }
 
     var title = metadata.title.trim();
+    var titleNode = document.createTextNode(title || '');
+    var titleBdiNode = document.createElement('bdi');
+    titleBdiNode.appendChild(titleNode);
+
     var artist = metadata.artist.trim();
-    var track = [];
+    var artistNode = document.createTextNode(artist || '');
+    var artistBdiNode = document.createElement('bdi');
+    artistBdiNode.appendChild(artistNode);
 
-    if (title) {
-      track.push(title);
-    }
-    if (artist) {
-      track.push(artist);
-    }
-    track = track.join(' — '); // Using a &mdash; here.
-    if (track) {
+    var emDashNode = document.createTextNode(' — '); // Using a &mdash; here.
+
+    if (title || artist) {
       this.track.removeAttribute('data-l10n-id');
-      this.track.textContent = track;
+      this.track.innerHtml = '';
```

Comment:
The property name is innerHTML, not innerHtml.  But you should use textContent anyway. Anytime you use innerHTML, you're adding code that needs to be audited for script injection attacks. Clearly using the empty string here is safe, but it is even safer to use textContent.